libcontainer/seccomp/config.go

```diff
@@ -66,6 +66,7 @@ var archs = map[string]string{
 	"SCMP_ARCH_PPC64LE":     "ppc64le",
 	"SCMP_ARCH_S390":        "s390",
 	"SCMP_ARCH_S390X":       "s390x",
+	"SCMP_ARCH_RISCV64":     "riscv64",
```

Comment:
We should probably sort this list as well, but looks like it's already a bit "unsorted"; perhaps move it before the `s390x` variants though (somewhat more consistent with the other lists)

Reply:
done